internal/config/config.go

```diff
@@ -45,6 +58,8 @@ var (
 // Runtime is the runtime configuration of the application.
 var Runtime *Config
 
+const goerliStarknetGateway = "http://alpha4.starknet.io"
+
 const goerli = "http://alpha4.starknet.io"
 
 func init() {
```

Comment:
Juno's default `DataDir` does not work on my machine. To prevent additional portability issues, we should follow [the XDG Base Directory spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html), which states:
> $XDG_DATA_HOME defines the base directory relative to which user-specific data files should be stored. If $XDG_DATA_HOME is either not set or empty, a default equal to $HOME/.local/share should be used.

Reply:
That's ok to solve, but isn't in the scope of this issue